docs/update-locking.md

```diff
@@ -16,9 +16,33 @@ assumption that updates are locked (see [issue #20](https://github.com/resin-io/
 
 In supervisors >= v4.0.0 and any OS that is not Resin OS 1.X, the old lock location is completely ignored.
 
-### Taking the lock
+### Creating the lockfile
 
-Using the above-mentioned library, the lock can be acquired like in this CoffeeScript example:
+There are many different tools and libraries to provide proper lockfile functionality. Here are a few examples, and you can search online for others in other programming languages not mentioned here explicitly.
```

Comment:
NIT
- `libraries to provide proper lockfile functionality` --> `libraries that provide proper lockfile functionality`
- Being able to search online for information is implicit, so we probably don't need to mention it, I'd reword this section as:
```
There are many different tools and libraries to provide proper lockfile functionality and a few common examples are shown below.
```

Reply:
Updated, thanks!